src/event_loop.h

```diff
@@ -57,4 +57,6 @@ class EventLoop {
 
   static void dispatchCallback(evutil_socket_t fd, short flags, void* param);
   void dispatchCall();
+  void pushCallList(const std::function<void()>& func);
+  void popCallList(std::function<void()>& func);
```

Comment:
One small change and this is good to go, "func" on this line to change to "call" to match the .cpp file and we are all good.

Reply:
I've tested, everything is working great.